woocommerce/woocommerce-ios
# Remove duplicated actions and models for OrderCount and OrderCountItem

Depends on https://github.com/woocommerce/woocommerce-ios/pull/4251

## Description

This PR removes Yosemite, Storage and Networking implemenations for:
- `countProcessingOrders` action
- `countOrders` remote method 
- `OrderCount` model
- `OrderCountItem` model

They are replaced by [OrderStatus](https://github.com/woocommerce/woocommerce-ios/blob/develop/Networking/Networking/Model/OrderStatus.swift) model and [retrieveOrderStatuses](https://github.com/woocommerce/woocommerce-ios/blob/bf0d731f9278a81718b1fb4a203d31f10e6b1f6a/Yosemite/Yosemite/Stores/OrderStatusStore.swift#L37) action.
More reasoning behind this change can be found in https://github.com/woocommerce/woocommerce-ios/pull/4251

## Test

1. Build and launch app from `develop`
2. Switch to this branch
3. Build and launch app
4. Check that app launched successfully and orders badge is correct

Update release notes:
